src/databroker_elasticsearch/__init__.py

```diff
@@ -8,3 +8,86 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+
+def callback_from_config(config):
+    """Construct ElasticCallback instance from a configuration dictionary.
+
+    Parameters
+    ----------
+    config : dict
+        The dictionary which contains "databroker-elasticsearch" key.
+
+    Returns
+    -------
+    ElasticCallback
+        The new instance of ElasticCallback.
+    """
+    from databroker_elasticsearch.elasticcallback import ElasticCallback
```

Comment:
why hide the import?

Reply:
This is to delay import of bluesky (which imports matplotlib and possibly a kitchen sink) until truly needed.   Import of `.elasticdocument.ElasticDocument` is light, fast and works without bluesky presence.

Reply:
You might consider getting ``CallbackBase`` which has been renamed to ``DocumentRouter`` here: https://github.com/NSLS-II/event-model/blob/master/event_model/__init__.py#L33

``event_model`` has fewer bulky imports

Reply:
Sounds good.
@danielballan - can you please confirm `DocumentRouter` is a correct base for bluesky callbacks - rather than `bluesky.callbacks.CallbackBase`?

Reply:
I would move forward with this change.